scripts/install_depthai.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+#--- Script for installing Kimera (RPGO) ---#
+
+
+### Setup ###
+
+## Source the default values for imported variables
+. scripts/default_values.sh
+
+## Source the global helper functions
+. scripts/system_utils.sh
+
+## Source the configurations for our dependencies
+. deps.sh
+
+
+### Package metadata setup ###
+
+if [ -z "${dep_name}" ]; then
+    dep_name="${dep_name_depthai}"
+fi
+
+if [ -z "${src_dir}" ]; then
+    src_dir="${parent_dir_depthai}/${dep_name_depthai}"
+fi
+
+if [ -z "${dep_ver}" ]; then
+    dep_ver="${dep_ver_depthai}"
+fi
```

Comment:
We are moving to this form now (for readability, starting in #270):
`dep_name="${dep_name:=dep_name_depthai}"`

Reply:
Is the format backwards compatible or do I need to wait for that to be implemented?

Reply:
Yup it's backwards compatible! We can change it here or wait until #270 with no issue